teams/teams-ps/teams/Set-CsOnlineLisCivicAddress.md

```diff
@@ -33,6 +33,11 @@ Set-CsOnlineLisCivicAddress -CivicAddressId <Guid> [-CompanyName <String>] [-Com
 ## DESCRIPTION
 Use the `Set-CsOnlineLisCivicAddress` cmdlet to modify an existing civic address which has not been validated. Validated civic addresses cannot be modified.
 
+Address editing is restricted to the following countries/regions:
+AU, BR, CA, HR, CZ, EE, HK, HU, IL, JP, LV, LT, MX, NZ, PL, PR, RO, SG, KR, SI, ZA, US
+
+If the user runs the PATCH request on one of the unsupported countries, it may interfere with number assignment and potentially is against regulatory requirements, so public use of the API is limited to the above countries.
```

Comment:
there is no PATCH request here. we should mention it as cmdlet.

Reply:
@vivekiiit  For the unsupported countries are we throwing any error?

Reply:
@duaneha-MS no, because we don't have an oce specific patch API